Modix.Data.Test/Sandbox.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+using NUnit.Framework;
+using NSubstitute;
+using Shouldly;
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+using Newtonsoft.Json;
+
+using Modix.Data;
+using Modix.Data.Models;
+using Modix.Data.Models.Core;
+using Modix.Data.Models.Moderation;
+using Modix.Data.Repositories;
+
+namespace Modix.Data.Test
+{
+    [TestFixture]
+    public class Sandbox
+    {
+        [Test]
+        public async Task Main()
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<ModixContext>();
+
+            var loggerFactory = Substitute.For<ILoggerFactory>();
+            loggerFactory.CreateLogger(Arg.Any<string>()).Returns((x) => new ConsoleLogger());
+
+            optionsBuilder.UseNpgsql("Server=localhost;Port=5432;Database=postgres;User Id=X;Password=X;");
```

Comment:
Why not use the in-memory database provider?

Reply:
Cause I don't know how to set it up. Really, that file doesn't need to be committed, it's just there for fucking around.